app/javascript/components/journey/overview/learning-section/TrackSummary.tsx

```diff
@@ -69,7 +69,7 @@ export const TrackSummary = ({
             </strong>{' '}
             in progress and{' '}
             <strong>
-              {track.numInProgressMentoringDiscussions === 0
+              {track.numQueuedMentoringRequests === 0
```

Comment:
I haven't yet added a test for this. Let me know if you'd like me to

Reply:
I'm fine without here :)